src/ts/editusers.ts

```diff
@@ -54,6 +92,19 @@ document.addEventListener('DOMContentLoaded', () => {
         ApiC.post('users', {...user});
       });
 
+    // UNARCHIVE AND ADD TO TEAM
+    } else if (el.matches('[data-action="unarchive-and-add-to-team"]')) {
+      ApiC.patch(`${Model.User}/${el.dataset.userid}`, {action: Action.Lock}).then(() => {
+        const params = {action: Action.Add};
+        if (el.dataset.team) {
+          params['team'] = el.dataset.team;
+        }
+        ApiC.patch(`${Model.User}/${el.dataset.userid}`, params).then(() => {
+          document.getElementById('archivedUsersFound').remove();
```

Comment:
```suggestion
          document.getElementById('archivedUsersFound').remove();
          // re-enable create button
          document.querySelector('[data-action="create-user"]').removeAttribute('disabled');
```

the button remains disabled so we cannot retry or create another user after getting any fail
![if user exists button is still invalid so can not create another user](https://github.com/user-attachments/assets/78a98d91-5daa-4e41-85cf-986b448f3f7f)